rpcs3/Emu/Cell/PPUTranslator.cpp

```diff
@@ -345,16 +345,23 @@ void PPUTranslator::CallFunction(u64 target, Value* indirect)
 
 	if (!indirect)
 	{
-		if ((!m_reloc && target < 0x10000) || target >= 0x100000000u - 0x10000)
+		const u64 base = m_reloc ? m_reloc->addr : 0;
+		const u32 caddr = m_info.segs[0].addr;
+		const u32 cend = caddr + m_info.segs[0].size - 1;
+		const u64 _target = target + base;
+
+		if (_target >= caddr && _target <= cend)
```

Comment:
Is it possible for a legit module to have more than 1 segment? Or is a far jump between segments disallowed?

Reply:
As a follow up to that, is it possible to be running outside seg[0]? Because this check seems very narrow.

Reply:
PPU Analyzer currently assumes segement 0 is the only executable segment.